tests/unit/test__opentelemetry_tracing.py

```diff
@@ -11,9 +11,16 @@
 
 from google.api_core.exceptions import GoogleAPICallError
 from google.cloud.spanner_v1 import _opentelemetry_tracing
+from google.cloud.spanner_v1 import gapic_version
 
-from tests._helpers import OpenTelemetryBase, HAS_OPENTELEMETRY_INSTALLED
+from tests._helpers import (
+    OpenTelemetryBase,
+    HAS_OPENTELEMETRY_INSTALLED,
+    OTEL_SCOPE_NAME,
+    OTEL_SCOPE_VERSION,
```

Comment:
In this file you can directly import from from opentelemetry.semconv.attributes import (
       OTEL_SCOPE_NAME,
       OTEL_SCOPE_VERSION,
) instead of helpers

Reply:
See that's a confusing request because just right above that line there is a try statement to try to import opentelemetry and if it fails there is a conditional to do nothing if the import fails
https://github.com/googleapis/python-spanner/blob/a941adb987ec6e2d1a1583a5aa1254908292e354/tests/unit/test__opentelemetry_tracing.py#L6-L15

and in the helper, there is an explicit polyfilling for those variables if any import failed as we do for other variables
<img width="563" alt="Screenshot 2024-09-13 at 5 03 39 AM" src="https://github.com/user-attachments/assets/0f00b7b4-b6c5-4a97-865a-0f437db96d64">

so it is much more nuanced than just seeming like a duplicate import

Reply:
Thinking out loud,
The polyfill that is used in _helpers.py is only for `HAS_OPENTELEMETRY_INSTALLED` to understand whether opentelemetry is installed or not. 
With your approach where you are hardcoding the 2 attributes `OTEL_SCOPE_NAME,
OTEL_SCOPE_VERSION ` in the catch block, these hardcoding will never get used because when 
1. opentelemetry is not installed then all tests skip. 
2. If opentelemetry is present then the attributes are directly used from semconv. 
 
There is no scenario where a hardcoded attribute will get used.

So, my point is the semconv attributes can be directly imported inside the try block of `tests/unit/test__opentelemetry_tracing.py` similar to `opentelemetry.trace.status import StatusCode` and used where needed.

Reply:
Alright, thank you @harshachinta done